### ISSUE TITLE
Color not applied on first item of itemize inside a block

### ISSUE THREAD

**Issue by valdre** [josephwright/beamer#386]
Dear Maintainer,
since February I'm encountering an issue when I use a list (itemize or enumerate) inside a block. I'm using Beamer version 2017/01/22 v3.40 from the Debian package with pdfTeX 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian) on Debian "stretch" testing release. I don't know which version of beamer I had before the issue appeared. I'm compiling with pdflatex, but I tried also with latex+dvips+ps2pdf and I found the same result. The problem is the following: it's impossible to change the color of the first element of a list inside a block. In particular I noticed that:
* The issue is present only if the list is the first thing in the block
  * I'm using a horrible workaround by adding an empty line and then removing an arbitrary amount of space before the list (thus the list is not the first thing anymore)
* The issue is not present if the list is not inside a block
* The issue is present only with some themes (e.g. Frankfurt)
* The issue affects both \color{} and \alert<>{} commands
I attach a test file to highlight the problem and my pdf output. I had to rename test.tex to test.txt because ".tex" file loading wasn't supported:
[test.txt](https://github.com/josephwright/beamer/files/875028/test.txt)
[test.pdf](https://github.com/josephwright/beamer/files/875019/test.pdf)

Thanks and Best Regards,
Simone Valdré

**Comment by josephwright** [comment]
This should be fixed in the latest release (v3.41): please make sure your TeX system is up-to date.

**Comment by valdre** [comment]
I apologize for opening this issue; in fact before opening it I tried to use the latest beamer.cls file from CTAN, but I did not use all the other files of the beamer package. Now, after updating everything to the latest version, the issue is not present anymore. Unfortunately there is not the latest version of beamer in any Debian repository (even in unstable or experimental). I hope that they'll update it soon! I'm really sorry for the false alarm.
Thanks and Best Regards,
Simone Valdré
